docs/src/languages/php/swoole.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Swoole"
+weight: 8
+sidebarTitle: "Swoole"
+---
+
+Swoole is a PHP extension that extends PHP core with a coroutine based asynchronous network application framework designed for building large scale concurrent systems.
+
+Unlike PHP-FPM’s stateless operating, Swoole relies on establishing persistent connections with every user, sending and receiving data in real-time.
+
+[Swoole](https://github.com/swoole/swoole-src) and [Open Swoole](https://openswoole.com/) are two forked libraries persuing that goal.
+
+{{< note >}}
+Swoole requires PHP 7.3+.
+{{< /note >}}
+
+Check the documentation related to [Laravel Octane on Platform.sh](../../guides/laravel/deploy/octane.md).
```

Comment:
@CollierCZ I've also added a link to the Octane installation doc page as most Swoole users seem to be using Laravel.